config/routes.rb

```diff
@@ -24,7 +22,6 @@
   get '/users/:id/sessions' => 'sessions#userIndex'
 
 
-
   #index services under the sun
```

Comment:
you guys should use more [resources](http://guides.rubyonrails.org/routing.html#crud-verbs-and-actions) for routing... it will make life easier to consume the API as it will be more restful...